ext/enchant/enchant.stub.php

```diff
@@ -2,94 +2,72 @@
 
 /** @generate-function-entries */
 
-/** @return resource|false */
-function enchant_broker_init() {}
+final class EnchantBroker
+{
+}
 
-/** @param resource $broker */
-function enchant_broker_free($broker): bool {}
+final class EnchantDict
+{
+}
 
-/**
-* @param resource $broker
-* @return string|false
-*/
-function enchant_broker_get_error($broker) {}
+function enchant_broker_init(): EnchantBroker|false {}
+
+function enchant_broker_free(EnchantBroker $broker): bool {}
+
+function enchant_broker_get_error(EnchantBroker $broker): string|false {}
 
 /**
-* @param resource $broker
 * @deprecated
 */
-function enchant_broker_set_dict_path($broker, int $name, string $value): bool {}
+function enchant_broker_set_dict_path(EnchantBroker $broker, int $name, string $value): bool {}
 
 /**
-* @param resource $broker
 * @deprecated
 */
-function enchant_broker_get_dict_path($broker, int $name): string|false {}
+function enchant_broker_get_dict_path(EnchantBroker$broker, int $name): string|false {}
```

Comment:
nit: missing space (same for the two stubs below)
```suggestion
function enchant_broker_get_dict_path(EnchantBroker $broker, int $name): string|false {}
```

Reply:
indeed, good catch